build-local.ps1

```diff
@@ -18,7 +18,7 @@ Write-Host "
 ##################################################################################################
 " -ForegroundColor Cyan
 
-./build.ps1 -BuildVerbosity Minimal -Verbosity Minimal -PackInParallel -AppendTimestamp -SetOctopusServerVersion 
+./build.ps1 -BuildVerbosity Minimal -Verbosity Normal -PackInParallel -AppendTimestamp -SetOctopusServerVersion 
```

Comment:
Changing this to Normal Verbosity. It's for local builds, but if we want to keep it Minimal, I'm happy to change this back.

Reply:
I prefer being able to see some detail. The `Minimal` was a bit useless when things went wrong.